src/js/modules/GET-load-likes-home.js

```diff
@@ -0,0 +1,19 @@
+import loadBreeds from './load-counter-home';
+
+export const GETLikes = async () => {
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/aliNSyHfj9kbhiXrqKsy/likes/', {
+    method: 'GET',
+  });
```

Comment:
Maybe we should keep the keys and URLs in one file after we merge all the code (as opposed to having it hardcoded in several places). Do you want to make an issue about that?

Reply:
Yeah, that's a very good idea!